src/meta/processors/admin/BalanceTask.h

```diff
@@ -73,7 +75,17 @@ class BalanceTask {
 
     void rollback();
 
-    Result result() const {
+    void markInvalidIfNotStarted() {
```

Comment:
maybe named markStopWhenStarted is better.

Reply:
Actually, the Status::START means it is not stated yet, so I rename it `markStopIfNotStarted`